lighthouse-core/scripts/release/prepare-commit.sh

```diff
@@ -17,7 +17,7 @@ fi
 
 OLD_VERSION=$(node -e "console.log(require('./package.json').version)")
 NEW_VERSION=$1
-BRANCH_NAME="bump_$NEW_VERSION"
+BRANCH_NAME="v$NEW_VERSION"
```

Comment:
will this conflict with actual tags in any way? I always avoided naming my branch something that I plan to tag so it's always unambiguous but it could just be a git boogeyman I inherited

Reply:
nothing technical. this branch is ephemeral so there shouldn't be any confusion, you should never even need to know about it.

Reply:
> nothing technical. just in your head :)

LIES!

```
➜  git-tag git init
Initialized empty Git repository
➜  git-tag git:(master) touch foo
➜  git-tag git:(master) ✗ git add -A
➜  git-tag git:(master) ✗ git commit -m 'foo'
[master (root-commit) 6079296] foo
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 foo
➜  git-tag git:(master) git checkout -b v1.0.0
Switched to a new branch 'v1.0.0'
➜  git-tag git:(v1.0.0) touch bar                                                                                                                                                                     ➜  git-tag git:(v1.0.0) ✗ git add -A
➜  git-tag git:(v1.0.0) ✗ git commit -m 'bar'
[v1.0.0 cd7d050] bar
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 bar
➜  git-tag git:(v1.0.0) git checkout master
Switched to branch 'master'
➜  git-tag git:(master) git tag -a v1.0.0 -m 'version tag'
➜  git-tag git:(master) touch baz
➜  git-tag git:(master) ✗ git add -A
➜  git-tag git:(master) ✗ git commit -m 'baz'
[master 0df3345] baz
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 baz
➜  git-tag git:(master) git checkout v1.0.0
warning: refname 'v1.0.0' is ambiguous.
Switched to branch 'v1.0.0'
```

Reply:
oh no

Reply:
ok i'll just do the other thing in this pr.